src/components/common/form/InputRounded.js

```diff
@@ -59,7 +58,8 @@ const getStylesFromProps = ({ theme }) => {
     fontFamily: theme.fonts.default,
     fontSize: normalize(14),
     fontWeight: '400',
-    lineHeight: 36,
+    paddingVertical: Platform.select({ web: 0, default: 7 }),
+    lineHeight: Platform.select({ web: 36, default: 24 }),
```

Comment:
Why is the lineHeight different?

Reply:
36 is a excessive size for a mobile version because the input grows.